Foundation-Mission-Structure-and-Strategy-Proposal.md

```diff
@@ -0,0 +1,76 @@
+# Node.js Foundation Mission, Structure and Strategy
+
+_Proposal to the Node.js Foundation TSC and Board, by Rod Vagg_
+
+**Goal #1:** Agree on a basic statement of mission for the Foundation, upon which all execution, by each group within the Foundation, can be tested and justified.
+
+**Goal #2:** Agree on a strategy generation and execution structure, and the relationship between the main bodies within that structure, that can best achieve the mission of the Foundation.
+
+## Foundation Mission Statement
+
+**The Node.js Foundation exists to increase the adoption of Node.js.**
+
+This goal is achieved by:
+
+1. Maintaining the health of the core Node.js project
+2. Fostering and encouraging a healthy Node.js open source ecosystem
+3. Fostering and encouraging a healthy commercial ecosystem of training, products and services for Node.js
+
+_This is an executable form of the Node.js Foundation Purpose as presented in its by-laws, Section 1.4_
+
+### Fundamendals
+
+The proposed mission statement stands on a fundamental raison d’être: “increase the adoption of Node.js”
+
+Execution of this mission is divided into three prioritised areas: core, open source ecosystem, commercial ecosystem
+
+Node.js core remains the fundamental focus for the Node.js Foundation, all else builds on this
+
+**The Foundation should operate according to a broad strategy that encompasses the three areas of concern: core, open source ecosystem and commercial ecosystem**
+
+![Strategy Areas](https://cldup.com/cqJ9KZppcA.png)
+
+## Foundation Structure
+
+The Node.js Foundation is made up of three distinct arms:
+
+* The ***Board***
+* The ***Business Executive***
+* The ***Technical Executive***
+
+### The Board
+
+* Ultimate responsibility for the activities of the Foundation and the expenditure of its funds
+* Representation of financial donors (member companies), the Technical Executive via a seat for a TSC nominee and the Node.js user community via external board seat(s)
+* Maintain focus on the fundamental goal of the Foundation: _“increase the adoption of Node.js.”_
+* Delegates all technical execution to the Technical Executive, requiring approval of strategy that expands scope beyond Node.js core related concerns
+* Delegates all business execution to the Business Executive, requiring approval of strategy and budget
+* Delegates a shared responsibility for the Node.js open source ecosystem to the Technical Executive and Business Executive, requiring approval of strategy
+
+### The Technical Executive
+
+* Answerable to the Board via a Technical Steering Committee (TSC), a peak body that takes on an administrative role within the Technical Executive
+* Managed by an _Open Governance_ process, independent of the Board, that empowers individuals and companies investing materially in the technical development of the project
+* Independently manages execution strategy for _“maintaining the health of the core Node.js project"_
```

Comment:
Do we have a good definition of what the core Node.js project is that we could link to here? Or maybe write one up?